backend/src/controller/controller.ts

```diff
@@ -37,6 +31,26 @@ export const getAllDoctorDetail = async (req: Request, res: Response) => {
   }
 };
 
+export const getIndividualDoctor = async (req: Request, res: Response) => {
+  let id = Number(req.params.id);
```

Comment:
Generally a good idea to use `const` over `let` since mutability is a big source of bugs. In this case, it looks like you never reassign `id` so assigning it to a `const` also helps convey your intent and makes the code easier to read. For example, if it's a `const`, I can assume it won't change in the following code so it helps ease my mental load when reading it.

```suggestion
  const id = Number(req.params.id);
```